content/ember/v5/deprecate-import-env-from-ember.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Ember.ENV
+until: 6.0.0
+since: 5.10.0
+---
+
+
+Previously, `ENV` could be accessed via the `Ember` import:
+```js
+import Ember from 'ember';
+
+Ember.ENV
+```
+
+If needed, `ENV` can be imported:
+```js
+import MyEnv from '<my-app>/config/environment';
```

Comment:
This doesn't explain what to do in an addon that doesn't know `<my-app>`.

Reply:
Fair, this has been fixed.